homeassistant/components/shopping_list/__init__.py

```diff
@@ -292,7 +292,7 @@ def websocket_handle_items(hass, connection, msg):
 def websocket_handle_add(hass, connection, msg):
     """Handle add item to shopping_list."""
     item = hass.data[DOMAIN].async_add(msg["name"])
-    hass.bus.async_fire(EVENT)
+    hass.bus.async_fire("shopping_list_item_added", {"item_added": item})
```

Comment:
I suggest that instead of placing the event directly, you create global variables (or a dictionary) to make changes easier. For example, change `EVENT` to `EVENT_UPDATE`, and `EVENT_ITEM_ADD`.

Reply:
Still running the checks, but do you mean like my most recent commit? Definitely not remotely a programmer, so I couldn't get the syntax to pass since it references the local variable "item" without doing it like I did. Thanks for the help!